e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/change-your-answers/change-your-answers-about-goods-or-services.spec.js

```diff
@@ -0,0 +1,145 @@
+import { submitButton } from '../../../../pages/shared';
+import { aboutGoodsOrServicesPage, checkYourAnswersPage } from '../../../../pages/insurance/policy-and-export';
+import partials from '../../../../partials';
+import { FIELD_IDS, FIELD_VALUES, ROUTES } from '../../../../../../constants';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+import checkText from '../../../../helpers/check-text';
+import application from '../../../../../fixtures/application';
+import countries from '../../../../../fixtures/countries';
+
+const {
+  POLICY_AND_EXPORTS: {
+    CHECK_YOUR_ANSWERS,
+    ABOUT_GOODS_OR_SERVICES_CHANGE,
+  },
+} = ROUTES.INSURANCE;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      ABOUT_GOODS_OR_SERVICES: { DESCRIPTION, FINAL_DESTINATION },
+    },
+  },
+} = FIELD_IDS;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.policyTypeAndExports;
+
+const { summaryList } = checkYourAnswersPage;
+
+const assertChangePageUrl = (referenceNumber, fieldId) => {
+  const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ABOUT_GOODS_OR_SERVICES_CHANGE}#${fieldId}-label`;
+  cy.url().should('eq', expected);
+};
+
+const assertAnswersPageUrl = (referenceNumber, fieldId) => {
+  const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}#${fieldId}-label`;
+
+  cy.url().should('eq', expected);
+};
+
+const assertSummaryListRowValue = (fieldId, expected) => {
```

Comment:
can this be made into a cy command? might be useful elsewhere in the future?

Reply:
Already done 🚀